GAME/CONTROL.C

```diff
@@ -1045,9 +1046,35 @@ void RemoveRoomFlipItems(struct room_info* r)//1F938(<), 1FB4C(<) (F)
 	}
 }
 
-void FlipMap(int FlipNumber)
+void FlipMap(int FlipNumber) // (F)
 {
-	S_Warn("[FlipMap] - Unimplemented!\n");
+	struct room_info* r = room;
+	for(int i = 0; i < number_rooms; i++, r++)
```

Comment:
i must be declared at the very beginning.

Reply:
fixed